src/renderer/entities/transaction/lib/transactionService.ts

```diff
@@ -35,6 +35,18 @@ import {
 } from './common/types';
 import { walletUtils } from '../../wallet';
 
+class RawSigner implements Signer {
+  signature: HexString;
+
```

Comment:
Why do we have to use classes with signer? Is there no way to do it without it?

Reply:
Moved to a separate file
```javascript
const options: Partial<SignerOptions> = {
    era: api.registry.createType('ExtrinsicEra', unsigned.era),
    signer: new RawSigner(signature),
    assetId: unsigned.assetId,
    tip: unsigned.tip,
    nonce: unsigned.nonce,
    blockHash: unsigned.blockHash,
  };
```
`signer` expects something that implements `Signer` interface so the easiest way is to create a class
@Asmadek anyway, can discuss another solution